### ISSUE TITLE
Winfilebeat Default config with GL 5.0 and Sidecar 1.2

### ISSUE THREAD

**Issue by janheise** [Graylog2/collector-sidecar#454]
### Problem description

The default config generated for Winfilebeat lists some variables that create problems with a Sidecar V1.2.
It works without problems with the current Beta 1.3 version.

### Steps to reproduce the problem
Install GL 5 and sidecar 1.2 on Windows, create a Winfilebeat config.

### Environment
![Screenshot 2022-11-14 at 15 43 30](https://user-images.githubusercontent.com/1190735/201692094-e15f8112-51aa-439e-9d6c-07b78c765465.png)

* Sidecar Version: 1.2
* Graylog Version: 5.0
* Operating System: Sidecar on Windows
* Elasticsearch Version:
* MongoDB Version:

**Comment by mpfz0r** [comment]
Correct, only 1.3 sidecars report this variable into Graylog. If you migrate from an older Graylog version, this isn't an issue,
because the configs were generated before we introduced these variables.

So the manual fix for the user would be, not to use the suggested template config, but change the config to using
the old fixed directory instead of the generated spooldir.
e.g.
```
path:
  data: /var/lib/graylog-sidecar/collectors/filebeat/data
  logs: /var/lib/graylog-sidecar/collectors/filebeat/log
```
